src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/stores/memoryFormStoreFactory.test.js

```diff
@@ -9,7 +9,7 @@ jest.mock('../../stores/metadataStore', () => ({
     getSchema: jest.fn(),
 }));
 
-test('Create a MemoryFormStore with schema', (done) => {
+test('Create a MemoryFormStore with schema', () => {
```

Comment:
a test cannot return a promise and use the `done` callback at the same time anymore:
https://jestjs.io/blog/2021/05/25/jest-27#features-coming-with-breaking-changes